src/test/java/org/example/repository/CartItemRepositoryTest.java

```diff
@@ -0,0 +1,53 @@
+package org.example.repository;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.example.model.CartItem;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.jdbc.Sql;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class CartItemRepositoryTest {
+    private static final long CART_ITEM_ID = 1L;
+    private static final long SHOPPING_CART_ID = 1L;
+    @Autowired
+    private CartItemRepository cartItemRepository;
+
+    @Test
+    @DisplayName("Get cart item by id and shopping cart id")
+    @Sql(scripts = {
+            "classpath:db/cartItem-repository-tests/"
+            + "before/insert-books-to-books-table.sql",
+            "classpath:db/cartItem-repository-tests/"
+            + "before/insert-user-to-user-table.sql",
+            "classpath:db/cartItem-repository-tests/"
+            + "before/insert-role_user-to-role_user-table.sql",
+            "classpath:db/cartItem-repository-tests/"
+            + "before/insert-shoppingCart-to-shoppingCarts-table.sql",
+            "classpath:db/cartItem-repository-tests/"
+            + "before/insert-cart_item-to-cart_items-table.sql"
+                    }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
```

Comment:
You could consolidate multiple SQL statements into a single .sql file for both the setup and teardown steps, and then use a single @Sql annotation pointing to that consolidated file. This can help in keeping your test methods clean and make it easier to manage your SQL scripts

Reply:
I dessagree. Previous mentor (Ivan) told my both options are right.